### ISSUE TITLE
Default logging too verbose in modules

### ISSUE THREAD

**Issue by suyograo** [elastic/logstash#7330]
```
bin/logstash --modules cef
```
emits a ton of log messages at `info` level which is not really necessary and can cause the user to be alarmed.

```
[2017-06-05T22:12:44,070][INFO ][logstash.modules.importer] Attempting PUT {:url_path=>"_template/cef", :file_path=>"/tmp/logstash-6.0.0-alpha2-SNAPSHOT/modules/cef/configuration/elasticsearch/cef.json"}
[2017-06-05T22:12:44,165][INFO ][logstash.elasticsearchclient] HEAD http://localhost:9200/_template/cef [status:404, request:0.094s, query:N/A]
[2017-06-05T22:12:44,166][FATAL][logstash.elasticsearchclient] [404]
[2017-06-05T22:12:44,275][INFO ][logstash.elasticsearchclient] PUT http://localhost:9200/_template/cef [status:200, request:0.070s, query:n/a]
[2017-06-05T22:12:44,342][INFO ][logstash.modules.importer] Attempting PUT {:url_path=>".kibana/index-pattern/cef-*", :file_path=>"/tmp/logstash-6.0.0-alpha2-SNAPSHOT/modules/cef/configuration/kibana/index-pattern/cef.json"}
[2017-06-05T22:12:44,354][INFO ][logstash.elasticsearchclient] HEAD http://localhost:9200/.kibana/index-pattern/cef-* [status:404, request:0.012s, query:N/A]
[2017-06-05T22:12:44,355][FATAL][logstash.elasticsearchclient] [404]
[2017-06-05T22:12:44,443][INFO ][logstash.elasticsearchclient] PUT http://localhost:9200/.kibana/index-pattern/cef-* [status:201, request:0.088s, query:n/a]
[2017-06-05T22:12:44,444][INFO ][logstash.modules.importer] Attempting PUT {:url_path=>".kibana/config/5.4.0", :file_path=>nil}
[2017-06-05T22:12:44,447][INFO ][logstash.elasticsearchclient] HEAD http://localhost:9200/.kibana/config/5.4.0 [status:404, request:0.003s, query:N/A]
[2017-06-05T22:12:44,447][FATAL][logstash.elasticsearchclient] [404]
[2017-06-05T22:12:44,465][INFO ][logstash.elasticsearchclient] PUT http://localhost:9200/.kibana/config/5.4.0 [status:201, request:0.017s, query:n/a]
```

Can we please suppress this to `debug` to reduce the noise?

**Comment by suyograo** [comment]
@guyboertje can you take a look after the 5.5 backport please?

**Comment by guyboertje** [comment]
Fixed with #7367
